packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2998,14 +3027,14 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     if (parent != null) {
       assert(() {
         if (parent == this) {
-          throw FlutterError(
-            'A GlobalKey was used multiple times inside one widget\'s child list.\n'
-            'The offending GlobalKey was: $key\n'
-            'The parent of the widgets with that key was:\n  $parent\n'
-            'The first child to get instantiated with that key became:\n  $element\n'
-            'The second child that was to be instantiated with that key was:\n  $widget\n'
-            'A GlobalKey can only be specified on one widget at a time in the widget tree.'
-          );
+          throw FlutterError.fromParts(<DiagnosticsNode>[
+            ErrorSummary('A GlobalKey was used multiple times inside one widget\'s child list.'),
+            DiagnosticsProperty<GlobalKey>('The offending GlobalKey was', key),
+            parent.describeElement('The parent of the widgets with that key was'),
```

Comment:
I guess we don't need to qualify `describeElement` with `parent` here?

Reply:
This code looks consistent with the previous error message which was describing the parent. If the `parent` was removed the error would be describing a describing the widget with the key not its parent.
```dart
'The parent of the widgets with that key was:\n  $parent\n'
```
became 
```dart
parent.describeElement('The parent of the widgets with that key was'),
```